server/ec2spotmanager/views.py

```diff
@@ -378,20 +379,20 @@ def __handleConfigPOST(request, config):
 
 
 @deny_restricted_users
-def createConfig(request):
+def createEc2Config(request):
     if request.method == 'POST':
-        config = PoolConfiguration()
-        return __handleConfigPOST(request, config)
+        config = Ec2PoolConfiguration()
+        return __handleEc2ConfigPOST(request, config)
     elif request.method == 'GET':
-        configurations = PoolConfiguration.objects.all()
+        configurations = Ec2PoolConfiguration.objects.all()
```

Comment:
This is used to render the list of parent configurations. We should allow a parent configuration to be same or super type.